src/backend/aspen/app/views/phylo_runs.py

```diff
@@ -56,35 +59,19 @@ def start_phylo_run():
     all_samples: Iterable[Sample] = g.db_session.query(Sample).options(
         joinedload(Sample.uploaded_pathogen_genome, innerjoin=True),
     )
-    # Step 3 - Enforce AuthZ
-    all_samples = authz_sample_filters(all_samples, sample_ids, user)
 
-    # Step 4 - Create a phylo run & associated input rows in the DB
-    pathogen_genomes: MutableSequence[PathogenGenome] = list()
-    found_sample_ids = set()
-    for sample in all_samples:
-        pathogen_genomes.append(sample.uploaded_pathogen_genome)
-        found_sample_ids.add(sample.private_identifier)
-        found_sample_ids.add(sample.public_identifier)
-    if len(pathogen_genomes) == 0:
-        sentry_sdk.capture_message(
-            f"No sequences selected for run from {sample_ids}.", "error"
-        )
-        raise ex.BadRequestException("No sequences selected for run")
+    # Step 3 - Enforce AuthZ (check if user has permission to see private identifiers)
+    all_samples = authz_sample_filters(all_samples, sample_ids, user)
```

Comment:
I think it might be good to rename `all_samples` -- it's really `user_visible_samples`, right? I know you're following the previous convention, but it seems a little confusing that `all_samples` goes from meaning _actually all the samples_ a few lines above, to meaning all the samples _visible to the user_ making the request once the authz func is applied.